src/figuresClasses.ts

```diff
@@ -1,19 +1,97 @@
-export interface Figure {
+function roundToHundredths(x: number): number {
+  return Math.round(100 * x) / 100;
```

Comment:
![image](https://user-images.githubusercontent.com/57574816/179517047-243d956a-44d2-4fca-95f0-67805fd9cd7f.png)
```suggestion
  return Math.floor(100 * x) / 100;
```